zipline/data/_minute_bar_internal.pyx

```diff
@@ -37,10 +37,50 @@ def minute_value(ndarray[long_t, ndim=1] market_opens,
     return market_opens[q] + r
 
 def find_position_of_minute(ndarray[long_t, ndim=1] market_opens,
+                            ndarray[long_t, ndim=1] market_closes,
```

Comment:
Do we still need the older logic at all?  Why not make `find_position_of_minute` always be aware of half days instead of having two different methods?